test/unittest/unittest_nntrainer_layers.cpp

```diff
@@ -2251,8 +2251,7 @@ class nntrainer_EmbeddingLayer
 
   virtual void prepareLayer() {
     int status = setProperty("in_dim=50 |"
-                             "out_dim=8 |"
-                             "in_length=12");
```

Comment:
So, do you expect to set this from input layer ?

Reply:
`in_length` can be interpreted directly based on the shape of the input. So, it has been removed.